src/main/resources/sql/insert_data.sql

```diff
@@ -0,0 +1,10 @@
+insert into room(price, capacity, type_class)
+values (1800, 2, 'GRAND'),
+       (2100, 2, 'STANDARD'),
+       (1800, 1, 'STANDARD'),
+       (3100, 1, 'SUPERIOR'),
+       (3800, 2, 'SUPERIOR'),
+       (4100, 1, 'DELUXE'),
+       (4900, 2, 'DELUXE'),
+       (4200, 4, 'FAMILY'),
+       (3300, 3, 'TRIPLE')
```

Comment:
Maybe we can add more difference price like 18900.435 180.00 and other prices with point . ?